packages/mui-lab/package.json

```diff
@@ -76,6 +76,7 @@
     "@mui/base": "5.0.0-alpha.73",
     "@mui/system": "^5.5.2",
     "@mui/utils": "^5.4.4",
+    "@mui/x-date-pickers": "https://pkg.csb.dev/mui/mui-x/commit/af717e5a/@mui/x-date-pickers",
```

Comment:
Is this intended?

Reply:
For the presence of `@mui/x-date-pickers` yes (see https://github.com/mui/material-ui/pull/31984#discussion_r835225814)

For the Codesandbox link instead of NPM, it's temporary since the pickers have not yet been published on X.
But this PR is in draft, I don't intend to merge it with those Codesandbox packages.

Reply:
Ah, it is still a draft. My bad 😅